src/com/createsend/Clients.java

```diff
@@ -116,16 +119,95 @@ public String create(Client client) throws CreateSendException {
+    public PagedResult<SentCampaign> sentCampaigns() throws CreateSendException {
+        return sentCampaigns(1, 1000, "desc");
+    }
+
+    /**
+     * Gets a paged list of campaigns sent by the current client
+     * @param page The page number of results to get. Use <code>null</code> for the default (page=1)
+     * @param pageSize The number of records to get on the current page. Use <code>null</code> for the default.
+     * @param orderDirection The direction to order results by. Use <code>null</code> for the default.
+     * @return The paged campaigns returned by the api call
+     * @throws CreateSendException Thrown when the API responds with a HTTP Status >= 400
+     * @see <a href="http://www.campaignmonitor.com/api/clients/#getting_client_campaigns" target="_blank">
+     * Getting sent campaigns</a>
+     */
+    public PagedResult<SentCampaign> sentCampaigns(
+            Integer page, Integer pageSize, String orderDirection) throws CreateSendException {
+        return sentCampaigns("", "", null, page, pageSize, orderDirection);
+    }
+
+    /**
+     * Gets a paged list of campaigns sent by the current client
+     * @param sentFromDate Campaigns sent on or after the <code>sentFromDate/code> value specified will be returned.
+     *                     Must be in the format YYYY-MM-DD. If not provided, results will go back to the beginning of the client’s history.
+     *                     Use <code>null</code> for the default
+     * @param sentToDate Campaigns sent on or before the <code>sentToDate/code> value specified will be returned.
+     *                   Must be in the format YYYY-MM-DD. If not provided, results will include the most recent sent campaigns.
+     *                   Use <code>null</code> for the default
+     * @param tags An array of tags to filter sent campaigns.
+     *             Sent campaigns with all the tags specified will be returned.
+     *             Use <code>null</code> for the default
+     * @param page The page number of results to get. Use <code>null</code> for the default (page=1)
+     * @param pageSize The number of records to get on the current page.
+     *                 The value can be between 1 and 1000.
+     *                 Use <code>null</code> for the default.
+     * @param orderDirection The direction to order results by.
+     *                       The value can be ASC or DESC
+     *                       Use <code>null</code> for the default (DESC)
+     * @return The paged campaigns returned by the api call
+     * @throws CreateSendException Thrown when the API responds with a HTTP Status >= 400
+     * @see <a href="http://www.campaignmonitor.com/api/clients/#getting_client_campaigns" target="_blank">
+     * Getting sent campaigns</a>
+     */
+    public PagedResult<SentCampaign> sentCampaigns(
+            Date sentFromDate, Date sentToDate, String[] tags,

```

Comment:
Should `tags` be a string array? Or do we keep it consistent with the API endpoint and just have it be comma delimited (This is what we do in the C#, Python, Ruby PRs)

Reply:
Interesting.  In my opinion, the wrapper should offer a string array.  I believe that's the correct mapping between the comma-delimited parameter of the API and the client application.